ci/eval/release-checks.nix

```
@@ -0,0 +1,127 @@
+{
+  lib,
+  runCommand,
+  attrpathsSuperset,
+  nixpkgs,
+  singleSystem,
+  linkFarm,
+}:
+
+{
+  evalSystem,
+  chunkSize,
+  quickTest ? false,
+}:
+
+let
+  # no point in pulling in another builder for this
+  abortSource = runCommand "dont-use-nix-path-in-nixpkgs.nix" { } ''
+    echo 'builtins.abort "Illegal use of <nixpkgs> in Nixpkgs"' > "$out"
+  '';
```

"We have `echo ... > $out` at home!"

The `echo ... > $out` at home:

```suggestion
  abortSource = builtins.toFile "dont-use-nix-path-in-nixpkgs.nix" ''
    builtins.abort "Illegal use of <nixpkgs> in Nixpkgs"
  '';
```

---

lol, thanks!